py/objdeque.c

```diff
@@ -138,21 +138,17 @@ STATIC mp_obj_t deque_popleft(mp_obj_t self_in) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(deque_popleft_obj, deque_popleft);
 
-#if 0
```

Comment:
I don't know why this was disabled, but this method is needed by the asyncio implementation.

Reply:
This was disabled years ago here: https://github.com/micropython/micropython/commit/6c3faf6c17521940eb3ec48a8af3d28b513e2ba2
> Allow to compile without warnings by disabling deque_clear.

Not sure if the original reason is still relevant.